application/src/main/java/com/sanction/thunder/email/EmailModule.java

```diff
@@ -39,4 +53,42 @@ EmailService provideEmailService(AmazonSimpleEmailService emailService) {
     return new EmailService(emailService, fromAddress);
   }
 
+  @Singleton
+  @Provides
+  @Named("successHtml")
+  String provideSuccessHtml() {
+    try {
+      return Resources.toString(Resources.getResource(successHtmlPath), Charsets.UTF_8);
+    } catch (IOException e) {
+      throw new EmailException("Error reading file from path: " + successHtmlPath, e);
+    } catch (Exception e) {
```

Comment:
I prefer not to catch the general `Exception`. Can you explicitly catch `NullPointerException` or whatever else might be thrown?